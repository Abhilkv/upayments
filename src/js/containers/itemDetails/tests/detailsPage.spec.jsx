import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import DetailsPage from '../detailsPage';

configure({ adapter: new Adapter() });

describe('components/detailsPage', () => {
  it('Renders default options', () => {
    const component = shallow(
      <DetailsPage productData={{}} />
    );
    const tree = shallowToJson(component);
    expect(tree).toMatchSnapshot();
  });
});
