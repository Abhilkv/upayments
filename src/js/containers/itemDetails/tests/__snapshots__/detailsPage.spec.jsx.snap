// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/detailsPage Renders default options 1`] = `
<div
  className="container"
>
  <div
    className="contentbox"
  >
    <div
      className="detailsWrapper"
    >
      <div
        className="imageWrapper"
      >
        <img
          className="image"
          height={280}
        />
      </div>
      <div
        className="nameBlock"
      >
        <span
          className="name"
        />
        <span
          className="price"
        >
          $
        </span>
      </div>
    </div>
    <div
      className="divider"
    />
    <div
      className="descriptionBox"
    >
      <span
        className="title"
      >
        Description
      </span>
      <span
        className="description"
      />
    </div>
  </div>
</div>
`;
